src/js/parsers/captions/vttparser.js

```diff
@@ -124,6 +124,9 @@ define(['parsers/captions/vttcue'], function(VTTCue) {
             callback(k, v);
         }
     }
+    
+    var defaults = new VTTCue(0,0,0);
+    var center = defaults.align === 'middle' ? 'middle' : 'center';
```

Comment:
Hi @mattcoz,

Can you add a comment here so it's clear why we need to determine the default align value?
 
Bugs have been filed with [Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=617989) and [Safari](https://bugs.webkit.org/show_bug.cgi?id=158478), so eventually this check won't be necessary. 

The WebVTT spec was updated here https://github.com/w3c/webvtt/pull/244 and, so far, FF is the only browser that matches the spec.